packages/cli/bin/cli.js

```diff
@@ -51,12 +52,34 @@ const notifier = updateNotifier({
 
 const i18nKey = 'commands.generalErrors';
 
-const CLI_UPGRADE_MESSAGE =
+const CMS_CLI_PACKAGE_NAME = '@hubspot/cms-cli';
+
+const CMS_CLI_UPGRADE_MESSAGE =
   chalk.bold('The CMS CLI is now the HubSpot CLI') +
-  '\n\nTo upgrade, run:\n\nnpm uninstall -g @hubspot/cms-cli\nand npm install -g @hubspot/cli';
+  `\n\nTo upgrade, uninstall ${chalk.bold(
+    CMS_CLI_PACKAGE_NAME
+  )} \nand then run ${uiCommandReference('{updateCommand}')}`;
+
+const CLI_UPGRADE_MESSAGE = `HubSpot CLI version ${chalk.cyan(
+  chalk.bold('{currentVersion}')
+)} is outdated. \nRun ${uiCommandReference(
+  '{updateCommand}'
+)} to upgrade to version ${chalk.cyan(chalk.bold('{latestVersion}'))}`;
```

Comment:
Should we `i18n` these messages?

Reply:
Good call, done in [7bda1dd](https://github.com/HubSpot/hubspot-cli/pull/1136/commits/7bda1dda97007a17c58d0015e9308185bd1433b6)